x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts

```diff
@@ -325,7 +325,7 @@ export const sortFieldOrUndefined = t.union([sort_field, t.undefined]);
 export type SortFieldOrUndefined = t.TypeOf<typeof sortFieldOrUndefined>;
 
 export const sort_order = t.keyof({ asc: null, desc: null });
-export type sortOrder = t.TypeOf<typeof sort_order>;
+export type SortOrder = t.TypeOf<typeof sort_order>;
```

Comment:
I think this was just a typo? Seems like all other typescript ones are uppercase.